edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Kubernetes.IntegrationTest/cluster/IKubernetesClientProvider.cs

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Agent.Kubernetes.IntegrationTest.Cluster
+{
+    using System.Threading.Tasks;
+    using k8s;
+
+    public interface IKubernetesClientProvider
```

Comment:
Why it is under IntegrationTest namespace? is it only used for testing purpose? why? do we have different implementation for this interface for testing?

Reply:
Yes, it is for tests only. Using this interface I intend to decouple Cluster manager (create, delete cluster and provide client) from an any cluster client provider.